cli/tests/pcluster/config/test_validators.py

```diff
@@ -404,6 +404,56 @@ def test_url_validator(mocker, boto3_stubber, capsys):
         utils.assert_param_validator(mocker, config_parser_dict, capsys=capsys, expected_warning=expected_message)
 
 
+@pytest.mark.parametrize(
+    "config, num_calls, bucket, expected_message",
+    [
+        (
+            {
+                "cluster default": {"fsx_settings": "fsx"},
+                "fsx fsx": {
+                    "storage_capacity": 1200,
+                    "import_path": "s3://test/test1/test2",
+                    "export_path": "s3://test/test1/test2",
+                    "auto_import_policy": "NEW",
+                },
+            },
+            2,
+            {"Bucket": "test"},
+            "AutoImport is not supported for cross-region buckets.",
+        ),
+    ],
```

Comment:
It would be nice to have test cases for the explicitly handled exception cases (e.g., the error code is "NoSuchBucket" or "AccessDenied").

You can force the boto3 stubber to generate client errors by adding a `generate_error` and `error_code` kwarg to the MockedBoto3Request constructor:
```python
MockedBoto3Request(
    method="get_bucket_location",
    response=get_bucket_location_response,
    expected_params=bucket,
    generate_error=error_code is not None,
    error_code=error_code,
)
```

You'll probably need to add an error_code parameter to your test cases for this as well.